index.html

```diff
@@ -0,0 +1,671 @@
+          <article class="d-flex flex-row">
+            <img class="img-fluid" src="img/side-1.png" alt="">
+            <a class="title-link" href="#"> The Sun Has Been Spotless for 28 Days Straight Despite New Solar Cycle</a>
+          </article>
+          <article class="middle-bar d-flex flex-row">
+            <img class="img-fluid" src="img/side-1.png" alt="">
+            <a class="title-link" href="#"> The Sun Has Been Spotless for 28 Days Straight Despite New Solar Cycle</a>
+          </article>
+          <article class="d-flex flex-row">
+            <img src="img/side-1.png" alt="">
+            <a class="title-link" href="#"> The Sun Has Been Spotless for 28 Days Straight Despite New Solar Cycle</a>
+          </article>
+        </div>
+      </div>
+    </section>
+    <section class="category mt-2 mb-4">
+      <div class="big-red-title">Sports</div>
+      <div class="row">
+        <div class="col-sm-12 col-md-4">
+          <img src="img/sports-1.jpg" alt="">
+          <a class="title-link" href="#"> NBA Playoffs: How to Watch LA Lakers vs Denver Nuggets on TV and Online</a>
+        </div>
+        <div class="col-sm-12 col-md-4">
+          <img src="img/sports-1.jpg" alt="">
+          <a class="title-link" href="#"> NBA Playoffs: How to Watch LA Lakers vs Denver Nuggets on TV and Online</a>
+        </div>
+        <div class="sidebar col-sm-12 col-md-4 d-flex flex-column">
+          <article class="d-flex flex-row">
+            <img class="img-fluid" src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+          <article class="middle-bar d-flex flex-row">
+            <img class="img-fluid" src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+          <article class="d-flex flex-row">
+            <img src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+        </div>
+      </div>
+    </section>
+
+    <section class="combo fluid-container mt-4">
+      <section class="row ml-3">
+        <section class="subscribe w-lg-100 col-lg-7 col-md-12 col-sm-12">
+          <span class="red-title"> SUBSCRIBE </span>
+          <div class="subscribe-box d-flex">
+            <img alt="Article Image" src="img/subscribe.png">
+            <div class="subscribe-content fluid-container d-flex">
+              <p>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</p>
+              <p>All Access + Weekly Delivery.</p>
+              <p>Choose A Membership That's Perfect for You!</p>
+              <p><button class="red-button"> SUBSCRIBE </button></p>
+            </div>
+          </div>
```

Comment:
[OPTIONAL] The original website title was definitely unreadable due to the background. Great job sticking with the existing styles. However, please change the title tag from p to h3-h4 and also increase its font-weight.